modules/making-disruptive-changes-br-ex-bridge.adoc

```diff
@@ -0,0 +1,120 @@
+====
+
+.Prerequisites
+
+* You created a manifest object that includes a `br-ex` bridge. 
+* You deployed your cluster that has the configured `br-ex` bridge.
+
+.Procedure
+
+. In an existing `MachineConfig` object, change parameter values for the NMState configuration. The following configuration enables the Dynamic Host Configuration Protocol (DHCP) on the `enp1s0` and the `enp2s0` network interface controllers (NICs) but disables DHCP on the `enp3s0` NIC.
++
+[IMPORTANT]
+====
+Before you save the `MachineConfig` object, check the changed parameter values. If you enter wrong values and save the file, you cannot recover the file to its original state and this impacts networking fucntionality for your cluster. 
+====
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 10-br-ex-master
+# ...
+networkConfig: &BOND
+  interfaces:
+    - name: enp1s0
+      type: interface
+      state: up
+      ipv4:
+        enabled: true
+        dhcp: true
+      ipv6:
+        enabled: false
+    - name: enp2s0
+      type: interface
+      state: up
+      ipv4:
+        dhcp: true
+        enabled: true
+      ipv6:
+        enabled: false
+    - name: enp3s0
+      type: interface
+      state: down
+      ipv4:
+        enabled: false
+      ipv6:
+# ...
+----
+
+// How to stop the Machine Config Operator (MCO) to applying system-level changes to these nodes.
+. Create an additional `MachineConfig` object and define basic configuration settings for control plane nodes and compute nodes in the object. The following example configuration defines some basic values for these nodes:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 10-force-reboot-master
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,
+        mode: 0644
+        overwrite: true
+        path: /etc/force-reboot
+---
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 10-force-reboot-worker
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,
+        mode: 0644
+        overwrite: true
+        path: /etc/force-reboot
+# ...
+----
+
+. Start a reboot operation by applying the changes from the file to your cluster:
++
+[source,terminal]
+----
+$ oc apply -f <additional_machine_config.yaml>
+----
++
+After the reboot operation, the changes made to the `MachineConfig` object that you created during cluster installation reflect in your cluster.
+// What if the changes don't reflect? Can we run any verification checks?
```

Comment:
Check the output of the nmstate and nmstate-configuration services.

Reply:
:heavy_check_mark: